sphinx_scality/layout.html

```diff
@@ -115,10 +115,13 @@ <h2><a href="{{ pathto(pagename) }}">{{ title }}</a></h2>
       <div class="sidebar-relinfo">
         {# TODO: add link to release notes and other hosted releases #}
         <p>Release: <span class="value">{{ release }}</span></p>
-      </div>
       {%- endif %}
+
+      {%- set has_releasenoteslink = theme_releasenoteslink | default('') != '' -%}
+      {%- if has_releasenoteslink %}<a href="{{ theme_releasenoteslink }}">{%- endif %} Release Notes {%- if has_releasenoteslink %}</a>{% endif %}
```

Comment:
Why would we want to keep the text without the link?

```suggestion
      {%- if has_releasenoteslink %}<a href="{{ theme_releasenoteslink }}">Release Notes</a>{% endif %}
```